<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings_Collection</class>
 <widget class="QWidget" name="Settings_Collection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true" extracomment="not translatable because not shown to the user">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Path to scan for music files:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CheckDirTree" name="dirTree"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_17">
     <item>
      <widget class="QCheckBox" name="enableEchonestCatalog">
       <property name="toolTip">
        <string>The Echo Nest supports keeping track of your catalog metadata
 and using it to craft personalized radios. Enabling this option
 will allow you (and all your friends) to create automatic playlists
 and stations based on your personal taste profile.</string>
       </property>
       <property name="text">
        <string>Upload collection list to The Echo Nest to enable user radio</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxWatchForChanges">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Watch for changes</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="scanTimeLabel">
         <property name="text">
          <string>Time between scans, in seconds:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="scannerTimeSpinBox">
         <property name="minimum">
          <number>60</number>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckDirTree</class>
   <extends>QTreeView</extends>
   <header location="global">widgets/CheckDirTree.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
